Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -70,8 +68,12 @@ protected override void OnApplyTemplate()
             base.OnApplyTemplate();
 
             _detailsPresenter = (ContentPresenter)GetTemplateChild(PartDetailsPresenter);
+            _masterList = (ListViewBase)GetTemplateChild(MasterList);
 
             SetMasterHeaderVisibility();
+
+            _masterList.ItemClick -= MasterListItemClick;
```

Comment:
That makes sense, although it would be nice since we can get a lot of the click/selection logic for free if we could go that route. What is a base type we can assume it to be?